### DISCUSSION TITLE
Can "Include Views" in the DDL be disabled?

### DISCUSSION THREAD

**Discussion by RadToy62** [dbeaver/dbeaver#23477] [General]
When I click on a table to view it's properties, and select the DDL tab, all of the views for the schema are displayed in the DDL windows, whether they are related to the table or not. 

Under the DDL window, there is a small window that reads "Include Views", I would like that to be aways off by default (as opposed to always on). I ran through the settings and didn't see a setting related to this option.

![includeviews](https://github.com/dbeaver/dbeaver/assets/161335730/8ec52570-dbdc-4fd2-ab03-0af26955491c)

**Comment by LonwoLonwo** [comment]
Hello @RadToy62 
No, we do not have such an option.
This button keeps its state, so if you click first, it will always be false or true.
So, it needs to be clarified now - why do you want to disable this button?

**Comment by RadToy62** [comment]
Hi,

Well, first of all, it seems to display all of the views for the schema, not just to ones that use the table that is being viewed.
Secondly, I generally am grabbing the DDL for the table to do something with it, and rarely ever need the view definition.

Lately I have been assigned to push all of the individual object DDL to GIT - tables, views, sp's, functions, etc... I need the views separate from the tables. I'll open the table ddl and do a save to a file, but I have to click on the view button to disable it from displaying. Since there is a way to enable/disable the display in the DDL panel, I was wondering if there was a global setting.

**Comment by RadToy62** [comment]
So, I was just playing around and the button only persists if you don't close the tab for the table or restart the tool.
I understand why you would do that, because if someone disables the views in the DDL, they might not realize they are missing.

The other issue with all of the views being displayed is no longer happening. I upgraded the software on Friday and I noticed the issue a couple weeks back.

I also got IBM Data Studio, which allows me to do more of what I need to do. I'm going to use that product for my project.
